docs/intro/tutorial06.txt

```diff
@@ -44,11 +44,11 @@ reference the path for templates.
 
 .. admonition:: Static file namespacing
 
-    Just like templates, we *might* be able to get away with putting our static
-    files directly in ``polls/static`` (rather than creating another ``polls``
-    subdirectory), but it would actually be a bad idea. Django will choose the
-    first static file it finds whose name matches, and if you had a static file
-    with the same name in a *different* application, Django would be unable to
+    We *might* be able to get away with putting our static files directly in
```

Comment:
It's because previously, you'd dealt with project-level templates already, in the admin template customizations. With the new ordering, you haven't yet touched on that, so it doesn't make much sense to reference it. I think the change is correct.